data/ibm_store_html/9.html

```diff
@@ -1,541 +1,541 @@
+
+<script type="text/javascript">
<!--
function swap(imgName, imgSrc) {
   var oimg=document.getElementById(imgName);
   oimg.src=imgSrc;
}

var myWindow;
function gotoPage(nbr) {
    var frm = document.forms[0];
    frm.elements['pnbr'].value = nbr;
    frm.submit();
    return false;
}
function IsAlpha(objevent,submitOnEnter) {
    if(window.event) { 
       if (window.event.type != 'change')
           key = window.event.keyCode
       else
	        key=objevent.charCodeAt();
    }else if (objevent) {
       key=objevent.which; 
    }else return true; 

    if (key==32 || (key>=39 && key<=41) || (key>=44 && key<=46) || key==96 || (key>=48 && key<=57) || (key>=65 && key<=90) || (key>=97 && key<=122)) {
       return true; 
    }else if (key==13 && submitOnEnter) {
      // They have choosen to hit enter, so lets submit the page.
      document.frmMain.submit();
      return false;
    }else return false; 
}
function IsNumber(objevent,submitOnEnter) {
    if(window.event) { 
       if (window.event.type != 'change')
           key = window.event.keyCode
       else
	        key=objevent.charCodeAt()
    }else if (objevent) {
       key=objevent.which;
    }else return true; 

    if (key==46 || (key>=48 && key<=57) || key==8 || key==0) {
       return true; 
    }else if (key==13 && submitOnEnter) {
      // They have choosen to hit enter, so lets submit the page.
      document.frmMain.submit();
      return false;
    }else return false; 
}
function DisableSubmitButtons () {
    theform=document.forms[0];
    if (document.all || document.getElementById) {
           for (i = 0; i < theform.length; i++) {
               var tempobj = theform.elements[i];
	            if (tempobj.type.toLowerCase() == "submit")
                   tempobj.disabled = true;
           }
    }
}
function raiseTab(tabId, maxTab) {
   var idArray = tabId.split('_');
   if (idArray.length == 2) {
       var productId = idArray[0];
       var selectedTabIndex = idArray[1];
       for(i=0;i<maxTab;i++) {
           var curTabId = "tab" + productId + "_" + i;
           var curPanelId = "panel" + productId + "_" + i;
           var curTab = window.document.getElementById(curTabId);
           var curPanel = window.document.getElementById(curPanelId);
           if(i==selectedTabIndex)  {
               curPanel.style.zIndex = 2;
               curPanel.style.visibility = 'visible';
               curTab.className  = "ActiveTab";
           }else {
               curPanel.style.zIndex = 0;
               curPanel.style.visibility = 'hidden';
               curTab.className  = "InactiveTab";
           } //end if
       } //end for 
   }
}
function ValidateMultiLineMaxLength(txtBox, MaxLength) {
var tmpMaxLength = MaxLength + 1
if(txtBox.value.length >= tmpMaxLength) {
txtBox.value = txtBox.value.substring(0, MaxLength); 
return false;}}
// -->
</script>
<script type="text/javascript">
```

Comment:
@dolph A check for ^M would be nice

Reply:
inconsistent carriage returns (DOS vs Unix) would be something CI should catch and we should fix.